### PR TITLE
Update and rename usernames.txt to usernames.md

### PR DESCRIPTION
**Pull request by chop0**

### CODE REVIEW

**Comment by ky28059** [comment]
I agree with this pull request.

File: usernames.md [hunk 1]
```diff
@@ -1 +1,2 @@
+# chop0
```
Comment (SheepTester) [inline usernames.md:1]: This rename is unnecessary

**Review by SheepTester — CHANGES_REQUESTED** [review]

**Comment by SheepTester** [comment]
@chop0 you have conflicts

**Comment by acrantel** [comment]
no

**Pull request closed without merging**